src/pages/ThorChainLP/Position/Position.tsx

```diff
@@ -407,10 +407,13 @@ export const Position = () => {
                   />
                 </TabPanel>
                 <TabPanel px={0} py={0}>
-                  <RemoveLiquidity
-                    headerComponent={TabHeader}
-                    opportunityId={foundPool.opportunityId}
-                  />
+                  {params.poolAccountId && (
+                    <RemoveLiquidity
+                      headerComponent={TabHeader}
+                      opportunityId={foundPool.opportunityId}
+                      poolAccountId={params.poolAccountId}
+                    />
+                  )}
```

Comment:
is there an issue having an empty tab panel?

Reply:
You're right, not ideal. Fixed in https://github.com/shapeshift/web/pull/6175/commits/55d61881b96485d99aa59c851bf113e41c291381.